lib/readers/csv.js

```diff
@@ -170,6 +148,30 @@ class CSVReader extends EventEmitter {
     } while (true);
   }
 
+  _parseFxaLine (fxaLine, timestamp) {
+    const [ uid, email, locale, createDate ] = this._splitLineBuffer(fxaLine);
+    return {
+      createDate: this._normalizeCSVDate(createDate),
+      email: email,
+      locale: this._normalizeCSVLocale(locale),
+      normalizedEmail: this._normalizeCSVEmail(email),
+      normalizedUid: this._normalizeCSVUid(uid),
+      timestamp,
+      uid,
+    };
+  }
+
+  _parseSalesforceLine (salesforceLine, timestamp) {
+    const [ uid, email ] = this._splitLineBuffer(salesforceLine);
+    return {
+      email,
+      normalizedEmail: this._normalizeCSVEmail(email),
+      normalizedUid: this._normalizeCSVUid(uid),
+      timestamp,
+      uid,
+    };
+  }
+
```

Comment:
Actually, this would be good to revisit. Returning an object it cleaner code-wise than a variable length array.

Reply:
I'll open a PR with these changes.